api/v1alpha1/podtypes/access_policy.go

```diff
@@ -22,11 +22,16 @@ type OutboundPolicy struct {
 	External []ExternalRule `json:"external,omitempty"`
 }
 
+// If you add both Namespace and NamespacesByLabel to an InternalRule,
+// Namespace takes presedence and NamespacesByLabel is omitted.
+// +kubebuilder:object:generate=true
 type InternalRule struct {
-	//+kubebuilder:validation:Optional
-	Namespace string `json:"namespace,omitempty"`
 	//+kubebuilder:validation:Required
 	Application string `json:"application"`
+	//+kubebuilder:validation:Optional
+	Namespace string `json:"namespace,omitempty"`
+	//+kubebuilder:validation:Optional
+	NamespacesByLabel map[string]string `json:"namespacesByLabel,omitempty"`
```

Comment:
nice! one last thing is maybe documenting what Namespace does and what NamespacesByLabel does, also adding this to the README

worth adding to readme seeing as we still don't have skiperator released to pkg.go.dev quite yet :)

Reply:
Sorted in the commit below